contracts/mocks/Airbro1155NftMint.sol

```diff
@@ -17,6 +17,7 @@ contract AirBro1155NftMint is ERC1155, Ownable {
     mapping(string => uint256) public idMap;
     mapping(uint256 => string) public lookupMap;
 
+    // solhint-disable-next-line no-empty-blocks
     constructor() ERC1155("https://ipfs.moralis.io:2053/ipfs/") {}
```

Comment:
What is up with this link? :)
also `setUri` shouldn't be possible multiple times.
I as a user would feel better if i knew that admin can set token uri only once, meaning that he can't change it in the future